main.py

```
@@ -376,6 +377,104 @@ async def metrics():
+        if not token:
+            query_params = dict(websocket.query_params)
+            token = query_params.get('token')
+        
+        if token:
+            try:
+                from app.core.security import verify_access_token
+                payload = verify_access_token(token)
+                if payload and payload.get("sub"):
+                    user_id = payload["sub"]
+                    logger.info("WebSocket authenticated", user_id=user_id, path=path, auth_method="subprotocol" if subprotocols else "query")
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)